runtime/onert/backend/cpu/ops/BinaryArithmeticLayer.cc

```diff
@@ -34,8 +34,12 @@ template <nnfw::cker::BinaryArithmeticOpType arithmetic_type, typename T>
 void eval(const IPortableTensor *lhs, const IPortableTensor *rhs, IPortableTensor *output,
           nnfw::cker::BinaryArithmeticOpParam op_params)
 {
-  const bool need_broadcast =
-      nnfw::cker::ProcessBroadcastShapes(getTensorShape(lhs), getTensorShape(rhs), &op_params);
+  bool need_broadcast = false;
+  if (arithmetic_type == nnfw::cker::BinaryArithmeticOpType::DIV)
+    need_broadcast = !HaveSameShapes(lhs, rhs);
```

Comment:
It would be better to fix `ProcessBroadcastShapes()` than change this to call `HaveSameShapes`. `need_broadcast` is used for calling functions of `cker`.

Reply:
It may not share `ProcessBroadcastingShapes()` implement because `DIV` operation comes from different tflite version. If we fix `ProcessBroadcastingShapes()`, `ProcessBroadcastingShapes()` may have operation checking and use different flow.

Reply:
I see, I just checked TFLite interpreter code. `ProcessBroadcastingShapes()` is older version. I s old version faster than new version of `v2.3.0`?

Reply:
@ragmani As I wrote in https://github.com/Samsung/ONE/issues/3811#issuecomment-673289378 yesterday, recent versions (`v2.3.0` and perhaps including `v2.2.0` and so on) use `!HavsSameShape`. I did not update the condition since I did not measure all models and am not sure for performance. It may be done by another issue. @hseok-oh, @periannath and I already discussed and choose this solution in #2153. This commit roll back to previous one before #3782 for `Div`.

Reply:
I got it.